float32 x_pos
float32 y_pos
float32 vx
float32 vy
float32 width
float32 length
bool brake_status
